q

```diff
@@ -0,0 +1,64 @@
+[33mb3059b6[m Refine WS for FOSSGIS 2019
+[33mbdecbcc[m Merge pull request #8 from buehner/fossgis-2019
+[33m4b590c7[m Update contents for FOSSGIS 2019
+[33m2c63e86[m Merge pull request #7 from dnlkoch/fix-broken-image-link
+[33m1d392c5[m Fix broken link
+[33mbd35def[m Merge pull request #6 from terrestris/preparation-fossgis-2018
+[33m3c6a118[m Update workshop contents to OSGeoLive 11.0
+[33mec40c1d[m Merge pull request #5 from dnlkoch/setup-script-fix
+[33m8b57f53[m Set ownership on extracted directories to user:user
+[33m0405387[m Merge pull request #4 from dnlkoch/update-fossgis-2017
+[33mb04e64e[m Minor updates (typos, headers in rest calls, dead links)
+[33mea9beba[m Update contents to OSGeoLive 10.0 and GeoServer 2.8.3
+[33m99fc0c8[m Update year and place in welcome notes
+[33m82086b1[m Cleanup and typos
+[33m4865f7c[m Use variable instead of hardcoded URL
+[33ma2122cf[m Use https URLs
+[33m3d2b680[m Add link to PDF version
+[33mfaecdd1[m Fix GWC chapter
+[33m0e5c24d[m Make better use of the richquotes plugin
+[33m53bf886[m Get rid of hardcoded GS versions
+[33ma4c37cb[m Sort authors and minors
+[33mf1a9d5b[m Introduce variable for OSGeoLive version
+[33m148187f[m Merge pull request #3 from terrestris/use-wget
+[33m31eb104[m Use `wget -q -O -` instead of `curl`
+[33m94a9ba7[m Fix URLs
+[33m1140a46[m Fix download URLs
+[33me432ced[m Update local maven repo to GS 2.8.2
+[33m1f182c6[m Merge pull request #2 from ahennr/enhancements
+[33m1828036[m Add removed resources to commit list as well
+[33m7ecb5ff[m fixed some typos
+[33m1bc713d[m revised troubleshooting chapter
+[33me32036a[m revised REST chapter
+[33m604a2e1[m changed GeoServer base URL to those of OSGeoLive
+[33m299fd8e[m reviewed basics chapter
+[33m69b31a1[m changed order of zip command params
+[33m75b5077[m ignore materials in build or gh-pages because of file size limits
+[33m21d9c00[m removed maven repo because file size too large for raw.github; specify folder to cleanup
+[33m20779ca[m show extended logs
+[33m234c88e[m remove every existing file
+[33md2770a2[m fix broken link
+[33m02a4d4a[m remove materials folder as well
+[33mff783a2[m Wrap codeblocks in xmp tags to allow gitbook variable use
+[33m6b6a1b7[m Remove unneeded plugins
+[33m35ae3ec[m Add workshop materials
+[33m1b53305[m Adjusted gitbook variables
+[33m0cc4729[m Fixed layout
+[33mb462bee[m Correct links
+[33meb2b460[m Read authors from gitbook variables
+[33m0da38f9[m Add JVM hint table
+[33m55ae126[m Update README
+[33m33a28cd[m Output to dev/null
+[33m7fac7e6[m CI changes
+[33mf98da16[m Make script executable
+[33m045124a[m ci test
+[33m5c47a15[m Update security token
+[33m3b8fd68[m Add simple SLD resource
+[33mc276890[m Remove git submodule init as no submodules present
+[33m9705be6[m Add travis config
+[33mcf64a13[m Initial upload
+[33md950069[m Initial commit
```

Comment:
I assume, this file isn't intended to be merged?

Reply:
Yes, this file should not be merged.